Sources/Networking/Backend.swift

```diff
@@ -182,4 +188,12 @@ extension Backend {
 
     }
 
+    static func createDefaultProductFetcher() -> PurchasedProductsFetcherType {
+        if #available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *) {
+            return PurchasedProductsFetcher()
+        } else {
+            return VoidPurchasedProductsFetcher()
```

Comment:
This is needed because we need to inject `PurchasedProductsFetcherType`, but `PurchasedProductsFetcher` isn't available. It won't be used if it's not available anyway. Can you think of a better way to do this?